helix-core/src/main/java/org/apache/helix/controller/pipeline/Pipeline.java

```diff
@@ -35,7 +35,8 @@
 
   public enum Type {
     DEFAULT,
-    TASK
+    TASK,
+    MANAGEMENT_MODE
```

Comment:
Discussed. We do need this MANAGEMENT_MODE type because we use the type name for the pipeline in our code. It makes the code consistent.
Meanwhile, a java doc comment is added for this type, making it clear that this type is meant to be used by Helix internally, not by external users.